packages/edit-widgets/src/components/layout/index.js

```diff
@@ -22,30 +26,45 @@ import Notices from '../notices';
 function Layout( { blockEditorSettings } ) {
 	const [ selectedArea, setSelectedArea ] = useState( null );
 	return (
-		<SlotFillProvider>
-			<DropZoneProvider>
-				<BlockEditorKeyboardShortcuts.Register />
-				<Header />
-				<Sidebar />
-				<Notices />
-				<div
-					className="edit-widgets-layout__content"
-					role="region"
-					aria-label={ __( 'Widgets screen content' ) }
-					tabIndex="-1"
-					onFocus={ () => {
-						setSelectedArea( null );
-					} }
-				>
-					<WidgetAreas
-						selectedArea={ selectedArea }
-						setSelectedArea={ setSelectedArea }
-						blockEditorSettings={ blockEditorSettings }
-					/>
-				</div>
-				<Popover.Slot />
-			</DropZoneProvider>
-		</SlotFillProvider>
+		<>
+			<BlockEditorKeyboardShortcuts.Register />
+			<SlotFillProvider>
```

Comment:
Is there any valid reason why all the wrapping components aren't included in `EditorSkeleton`?

Reply:
No particular reason. Initially the component was more about the layout of the page, that said, this is an interesting idea. Makes me wonder if it's EditorSkeleton or just another component.

Reply:
Good call, it might be a better fit as a wrapper component. It all depends really :)